.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -295,6 +331,13 @@ def get_input(question, default=""):
     config['institution_homepage'] = INSTITUTION_HOMEPAGE
     config['debugging_enabled'] = DEBUGGING_ENABLED
 
+    config['email_user'] = EMAIL_USER
```

Comment:
The email information should only be written to email.json, we don't need this here.